.github/workflows/main.yml

```diff
@@ -36,21 +42,92 @@ jobs:
         uses: docker/setup-qemu-action@v3 # https://github.com/marketplace/actions/docker-setup-qemu
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v3 # https://github.com/marketplace/actions/docker-setup-buildx
+      - name: Login to DockerHub
+        if: ${{ env.CAN_PUSH_IMAGE == 'true' }}
+        uses: docker/login-action@v3 # https://github.com/marketplace/actions/docker-login
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      ######################################################################################################
+      # prairielearn/workspace-desktop
+      - name: Check whether workspaces/desktop was modified
+        run: tools/check_path_modified.sh workspaces/desktop workspaces_desktop_modified
+      - name: Build and push prairielearn/workspace-desktop
+        if: ${{ env.workspaces_desktop_modified }}
+        uses: docker/build-push-action@v5 # https://github.com/marketplace/actions/build-and-push-docker-images
+        with:
+          context: workspaces/desktop
+          platforms: linux/amd64,linux/arm64
```

Comment:
Are the arm64 build succeeding for these images? I seem to recall that it was only postgres that caused the arm builds to fail so this may well be fine.

Reply:
Yep! I copied all this almost verbatim from `.github/workflows/images.yml` which has been building ARM64 images for everything but `prairielearn/plbase` and `prairielearn/prairielearn` for over two years now.